R/later.R

```diff
@@ -249,8 +256,10 @@ print.event_loop <- function(x, ...) {
 #'
 #' @export
 later <- function(func, delay = 0, loop = current_loop()) {
-  f <- rlang::as_function(func)
-  id <- execLater(f, delay, loop$id)
+  if (!is.function(func)) {
+    func <- rlang::as_function(func)
+  }
```

Comment:
Please add a comment explaining why the `if` is here -- otherwise someone who sees it in the future will probably be inclined to remove it.